### PR TITLE
[issue #567] Change the UI of time series chart

### PR DESCRIPTION
**Pull request by lengyyy**
## Overview
Please refer to issue #567.
It aims to change the UI of current time series chart in cloudberry:
1. from bar chart to **line** chart
2. from group by day to group by **week**
3. with time selection brush shown by default.

## Result

- Before:
image

- After:
by week
image

### CODE REVIEW

**Comment by baiqiushi** [comment]
Code looks good to me. But there's one issue I found here:
image
When I change the brush to a narrow time interval, the part outside selection is not dimmed, which was previous behavior.

**Comment by lengyyy** [comment]
@baiqiushi The "outside selection is not dimmed" problem has solved. Although the third-party library does not support, I implement this feature by js.

imageedit_1_5510314494

File: examples/twittermap/web/public/javascripts/timeseries/controllers.js [hunk 1]
```diff
@@ -110,21 +110,52 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
...
+            var resetClink = 0;
 
             var requestFunc = function(min, max) {
               cloudberry.parameters.timeInterval.start = min;
               cloudberry.parameters.timeInterval.end = max;
               moduleManager.publishEvent(moduleManager.EVENT.CHANGE_TIME_SERIES_RANGE, {min: min, max: max});
             };
 
+            var removeHighlight = function() {
+              var panel = $(".line")[0].parentElement;
```
[the remark this answers is not in the document]
Comment (baiqiushi) [reply examples/twittermap/web/public/javascripts/timeseries/controllers.js]: It'll be even better if you can avoid this `.line` class/style selection by using `#id` id selection on elements. I think it'll be safer.

**Comment by baiqiushi** [comment]
The argument is: we only need to show the brush explicitly at the beginning, once the user reset the timebar, it means the user already knows there's a brush feature there, at this time we can hide the brush though. I am OK with this idea.

**Pull request merged**